src/kubectl-helm-minikube/install.sh

```diff
@@ -15,6 +15,7 @@ rm -rf /var/lib/apt/lists/*
 KUBECTL_VERSION="${VERSION:-"latest"}"
 HELM_VERSION="${HELM:-"latest"}"
 MINIKUBE_VERSION="${MINIKUBE:-"latest"}" # latest is also valid
+TOOLS_PATH="${TOOLS_PATH:-"/usr/local/bin"}" # latest is also valid
```

Comment:
Can we create a test scenario to validate this new Feature option? thanks! 

 Ref: https://github.com/devcontainers/features/blob/7f1f31ff9e2b91cccb0b62567800c6a0f8a46a9c/test/kubectl-helm-minikube/scenarios.json

Reply:
Can we add some tests to validate this change?